testsuite/gnat2goto/tests/floating_point_literals/test.out

```diff
@@ -1,3 +1,5 @@
 [1] : FAILURE
+[3] : FAILURE
```

Comment:
This is a bit odd, normally I'd expect the successes and failures to come in order.

(Also, unrelated to this change: We will also need to get around to make the assertion format right for gnat2goto, it should read something like `[Floating_Point_Literals.assertion.1] assertion X > Z: SUCCESS`)